summingbird-online/src/main/scala/com/twitter/summingbird/online/option/AllOpts.scala

```diff
@@ -113,3 +113,9 @@ case class SummerParallelism(parHint: Int)
  * which are set by SummerParallelism.
  */
 case class SummerBatchMultiplier(get: Int)
+
+/**
+ * This option configures the flatMap to be merged into spout when set to true.
+ */
+case class FMMergeableWithSource(get: Boolean)
```

Comment:
can we move to the style a few options have of:
```scala
case class FMMergeableWithSource(get: Boolean)
object FMMergeableWithSource {
  def default: FMMergeableWithSource = FMMergeableWithSource(false)
}
```
So the default is close to the value?